### PR TITLE
Feat build task models

### PR DESCRIPTION
**Pull request by drewbo**
## Changes proposed in this pull request:
- Add build task DB models + migrations

## security considerations
None

### CODE REVIEW

File: api/models/build-task.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+        },
+      },
+    }, {
+      tableName: 'build_task',
+      paranoid: true,
+      indexes: [
+        {
+          name: 'build_task_build_id_type_index',
+          unique: true,
+          fields: ['buildId'],
```
Comment (apburnes) [inline api/models/build-task.js:47]: You need to add `buildTaskTypeId` based on the migration SQL.

**Review by apburnes — CHANGES_REQUESTED** [review]
You missed a field in the sequelize model of the build task index.

File: api/models/build-task.js (hunk 1 above)
Comment (drewbo) [reply api/models/build-task.js:47]: Excellent catch. My kingdom for not having to write these schemas twice

**Review by apburnes — APPROVED** [review]
🏗️

**Pull request merged**